modules/auxiliary/scanner/mssql/mssql_login.rb

```diff
@@ -34,6 +34,7 @@ def initialize
     )
     register_options([
       OptBool.new('TDSENCRYPTION', [ true, 'Use TLS/SSL for TDS data "Force Encryption"', false]),
+      OptBool.new('CreateSession', [false, 'Create a new session for every successful login', false])
```

Comment:
This might be a nitpick but should this be 
```suggestion
      OptBool.new('CreateSession', [false, 'Create a new session for every successful login', true])
```
making it required? just to keep it the same across the options and I don't think we'd want the value to ever be unset if it's been registered

Reply:
Ah, fair point. I'll get those updated 👍